src/components/metrics/card/index.js

```diff
@@ -118,6 +134,10 @@ export default class MetricCard extends Component {
     this.setState({sidebarIsOpen: (!this.state.sidebarIsOpen), modalIsOpen: false});
   }
 
+  _closeSidebar() {
+     this.state.sidebarIsOpen && this.setState({sidebarIsOpen: false})
```

Comment:
nit: I'd prefer to just use a normal if statement (though feel free to use a one liner) for control flow logic, I find it cleaner.